src/main/scala/org/ergoplatform/modifiers/history/Extension.scala

```diff
@@ -99,7 +107,7 @@ object ExtensionSerializer extends ScorexSerializer[Extension] {
     val startPosition = r.position
     val headerId = bytesToId(r.getBytes(Constants.ModifierIdSize))
     val fieldsSize = r.getUShort()
-    val fieldsView = (1 to fieldsSize).toStream.map {_ =>
+    val fieldsView = (1 to fieldsSize).toStream.map { _ =>
```

Comment:
@catena2w I suggest to rewrite this as imperative loop, both for efficiency and for code readability. 
Here we rely on two  implicit assumptions:
1) implementation of Stream.map execute in a right time (which we need to read from reader)
2) takeWhile do exactly required number of lambda invocations.

Reply:
In this PR I've just made a style fix of this line. Please create an issue for refactoring if you think it is important